.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,154 @@
+name: Automation - Prepare Release
+
+on:
+  workflow_dispatch:
+    # Determine the version number that will be assigned to the release. During the beta phase, we increment
+    # the minor version number and set the patch number to 0.
+    inputs:
+      candidate-stable:
+        required: true
+        description: Release candidate version (stable)
+      current-stable:
+        required: true
+        description: Current version (stable)
+      candidate-beta:
+        required: true
+        description: Release candidate version (beta)
+      current-beta:
+        required: true
+        description: Current version (beta)
+jobs:
+  # Releasing opentelemetry-collector
+  prepare-release:
+    runs-on: ubuntu-latest
+    steps:
+      # To keep track of the progress, it might be helpful to create a tracking issue similar to #6067. You are responsible
+      # for all of the steps under the "Performed by collector release manager" heading. Once the issue is created, you can
+      # create the individual ones by hovering them and clicking the "Convert to issue" button on the right hand side.
+      - name: Create issue for tracking release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          CANDIDATE_BETA: ${{ inputs.candidate-beta }}
+          CANDIDATE_STABLE: ${{ inputs.candidate-stable }}
+          REPO: codeboten/opentelemetry-collector
+        run: |
```

Comment:
How do you feel about having this in an actual source file?

Reply:
Unless we needed to re-use this code, I'm not sure if there's a value in having a separate script file, though I don't have a strong opinion on this.

Reply:
Being in a script means it could be used directly in case of necessity. It would also allow us to use a linter, like the shellcheck linter that is/was proposed as part of another PR.